scripts/globals/mobs.lua

```diff
@@ -744,3 +744,167 @@ xi.mob.difficulty =
     MAX                  = 8,
 }
 xi.mob.diff = xi.mob.difficulty
+
+-----------------------------------
+-- mob pet summoning
+-- If inactiveTime is > 0, will always use CAST_SUMMONER_START/STOP, otherwise will inject a mobskill action packet when the pets are summoned
+-----------------------------------
```

Comment:
This needs to have a much more detailed description. This seems like an edge case that bypasses the normal flow of monsters having other monsters as pets, so it should say something along those lines.

This latches pets onto their owner's `DEATH` and `ROAM_TICK` triggers, so it's already 50% towards counting as "entity iteration" - one of the big red no-no's for "this thing needs to be C++ only".